pygmt/src/histogram.py

```diff
@@ -51,8 +58,8 @@ def histogram(self, table, **kwargs):
     {aliases}
 
     Parameters
-    ----------
-    table : str or list or {table-like}
+    ---------
```

Comment:
Accidentally deleted a hyphen here I think
```suggestion
    ----------
```